azure-pipelines.yml

```diff
@@ -4,13 +4,40 @@
 
 name: $(TeamProject)_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
 
-variables:
-  MyRunNumber:  $[counter('MyRunNumber', 52201)]
-
+parameters:
```

Comment:
Move this to parameters so we can change it in azure pipeline UI

![image](https://github.com/dotnet/TorchSharp/assets/16876986/4a55d271-cc82-48a1-b9c3-ad10c7d5bd54)

Reply:
OK